module-system/sov-modules-api/src/lib.rs

```diff
@@ -141,7 +154,9 @@ pub trait Spec {
         + Serialize
         + for<'a> Deserialize<'a>
         + Send
-        + Sync;
+        + Sync
+        + 'static
```

Comment:
'static looks like very strict requirement, could you please elaborate, why it is needed?

Reply:
@citizen-stig, I think 'static is already enforced in practice by `for<'a> Deserialize<'a>`. 

But if we don't strictly need the extra bound, I'm all for removing it :)

Reply:
oh i already removed 'static requirement from a bunch of places. Needed it earlier, but we don't anymore. I think i missed this place. removed the bound and it works ok